app/src/main/java/com/glia/exampleapp/GliaWidgetsConfigManager.kt

```diff
@@ -26,6 +26,8 @@ object GliaWidgetsConfigManager {
     private const val REGION_KEY = "environment"
     private const val REGION_BETA = "beta"
     private const val REGION_ACCEPTANCE = "acceptance"
+    private const val BASE_DOMAIN = "base_domain"
+    private const val DEFAULT_BASE_DOMAIN = "at.samo.io"
```

Comment:
NIT: Do we need a default here? We have the same in core_sdk.

Reply:
Hm, maybe we can drop it.
I've implemented it the same way as REGION_ACCEPTANCE.
I would rather leave it with default values in both Widgets and Core.